components/SearchSandbox/template.js

```diff
@@ -79,6 +79,70 @@ const renderAsTree = (res, key = '0') => {
 };
 `;
 
+export function getComponentCode(config) {
```

Comment:
This looks good @jyash97 👍 

Can we use this same function to generate component code in `getApp()` method as well?